src/bci_build/package/python.py

```diff
@@ -98,8 +110,7 @@ def _get_python_kwargs(
         **_get_python_kwargs("3.11", os_version),
         package_name="python-3.11-image",
         support_level=SupportLevel.L3,
-        # https://peps.python.org/pep-0664/ defines 2027/10/31, SUSE offers until end of the year
-        supported_until=datetime.date(2027, 12, 31),
+        supported_until=_PYTHON_SUPPORT_ENDS["3.11"],
```

Comment:
Would it make sense to move this into `_get_python_kwargs`? Only if it makes the code simpler

Reply:
no, I think the supportlevel is product specific, and you would just end up doing product comparisons in `_get_python_kwargs`. you can do this here simpler.